strings/ion_laws.json

```diff
@@ -353,6 +359,7 @@
         "STUN BATONS",
         "SUITS",
         "SUNGLASSES",
+	"SUPERMATTER SHARDS",
```

Comment:
supermatter is plural and singular, it's like "sheep"
"get me the supermatter"
"pass me some supermatter"
"oh god, supermatter!"